### ISSUE TITLE
Release 1.1.2 - MacOS Installation Issues

### ISSUE THREAD

**Issue by Slothkar** [mintnick/eve-settings-manager#5]
Both the .dmg and .zip release files produce the same results: a popup prompting to move the app to the trash when attempting to run after downloading.

Sonoma 14.2.1 here if that's relevant.

![Screenshot 2024-01-31 at 17 54 35](https://github.com/mintnick/eve-settings-manager/assets/158848096/aa33098d-cbce-41e0-988c-4667ddb34c90)

**Comment by Bombe** [comment]
That’s because the binaries are not signed. Ctrl-click the icon in the Applications menu, you’ll get a popup that allows you to open it anyway.

**Comment by mintnick** [comment]
Thank you for the feedback. I just found the same problem.

TLDR: I uploaded a new dmg file in [release](https://github.com/mintnick/eve-settings-manager/releases/tag/v1.1.2), it should work as unidentified app instead of damaged app now.
If you would like to try it again and make it work, please let us know. Thank you!

A few steps you may need (on Sonoma):
1. Install from the dmg file
2. First time you open it, it says somthing like "cannot be opened because the developer cannot be verified." ( not "damaged" anymore)
3. **DON'T** "Move to Bin", select "Cancel"
4. Go to "System Settings" -> "Privacy & Security", scroll down, in Security section, there is a notification about the app
5. Choose "Open Anyway" -> another popup -> choose "Open"
6. First time it takes a few extra seconds to launch, it will be faster later

Other MacOS versions may have options in different places, but basically same process.

Explaination:
It seems to be a problem with Electron-builder (the building tool in this project) on newer MacOS. The official Electron documentation even recommand build on MacOS Rosetta.
Basically when app is built locally from source, it works. But if the file is downloaded (even with same file, to same computer), it is shown "damaged". And there is not even an option to allow it to run.

I didn't find a proper solution for this, so I ended up using Github Action to build it remotely on GitHub's server (whatever version they are using, it seems work), create a draft release from the build, download and re-upload the dmg file.
I will probably optimize the action file later to make it only run with certain tagged push.

**Comment by Slothkar** [comment]
> But if the file is downloaded (even with same file, to same computer), it is shown "damaged". And there is not even an option to allow it to run.

This is precisely the issue I was experiencing. Fortunately the new .dmg file installs just as it should, needing an exception to system security policy as you outlined.

Thanks for all the effort you have invested into this!
